src/components/Toolbar.js

```diff
@@ -5,11 +5,11 @@ export default class Toolbar extends Component {
     render() {
         return (
             <div>
-              <nav  className="navbar">
+                <nav  className="navbar">
```

Comment:
Delete the new spaces before the "<nav".
No need 2 spaces between "nav" and "className"